mlir/lib/Transforms/RemoveDeadValues.cpp

```diff
@@ -617,23 +719,33 @@ void RemoveDeadValues::runOnOperation() {
   auto &la = getAnalysis<RunLivenessAnalysis>();
   Operation *module = getOperation();
 
+  // Tracks values eligible for erasure - complements liveness analysis to
+  // identify "droppable" values.
+  DenseSet<Value> deletionSet;
+
+  // Maintains a list of Ops, values, branches, etc., slated for cleanup at the
+  // end of this pass.
+  CleanupList cl;
```

Comment:
Some people (myself included), associate `cl` with https://llvm.org/docs/CommandLine.html. Also, it's not a very descriptive. Perhaps `deadValsList`? Or just `deadVals`? But then there's more than just `Value`s 🤔 `finalCleanUpList`? Naming is hard.